ocw/lib/vault.py

```diff
@@ -63,7 +63,8 @@ def httpGet(self, path):
             else:
                 return r
         except Exception as e:
-            raise ConnectionError('Connection failed - {}: {}'.format(type(e).__name__, str(e)))
+            raise ConnectionError('URL: {} \n Response: {} \n {}: {}'.format(
+                self.url + path, r.status_code, type(e).__name__, str(e)))
```

Comment:
I would not use `r` uncondional here. 
From my point of view, we are trying to catch exceptions from `requests.get()` and then `r` isn't defined.

I didn't found the specific thing in python docs, but I gave this a try:

```
def throw_ex():
    raise Exception("FOOOOOO")
    return "bla"


try:
    foo = throw_ex()
except Exception as e:
    print(e)
    print(foo)
```
And if gave me:
```
FOOOOOO
Traceback (most recent call last):
  File "<stdin>", line 10, in <module>
NameError: name 'foo' is not defined
```

Reply:
yeah current approach is wrong I will try to find out something better ...